tests/TypeReconciliation/TypeTest.php

```diff
@@ -1710,7 +1710,7 @@ function returnsFalse() { return rand() % 2 > 0; }
 
                     if (count($a) < 1) {
                     }',
-                'error_message' => 'RedundantConditionGivenDocblockType'
+                'error_message' => 'DocblockTypeContradiction'
```

Comment:
This also feels like something that should be in Psalm 5.x

Reply:
I removed the BC break from this PR, this PR can now be directly merged in 5.x :)